src/PROPOSAL/PROPOSAL/secondaries/parametrization/photonuclear/Photonuclear.h

```diff
@@ -19,14 +19,22 @@ namespace secondaries {
             = PROPOSAL::InteractionType::Photonuclear;
         InteractionType GetInteractionType() const noexcept override { return type; }
 
-        std::vector<ParticleState> CalculateSecondaries(StochasticLoss, const Component&,
-                                                   std::vector<double>&) override
+        std::vector<ParticleState> CalculateSecondaries(
+                StochasticLoss loss, const Component&, std::vector<double>&) override
         {
             Logging::Get("proposal.Secondaries")
                     ->warn("PROPOSAL can not generate secondary particles"
                             "for a photonuclear interaction.");
             //TODO: Treatment for hadronic interactions
```

Comment:
if secondaries get returned, also the hadronic cascade should get returned; not just the lepton.
I am aware of the discussion in CORSIKA (see [here](https://gitlab.ikp.kit.edu/AirShowerPhysics/corsika/-/issues/411)), which can just handle real particles, but in my opinium the secondary output of PROPOSAl can also consist of hadronic pseudo particles containing the relevant information (Energy, position, direction, time and maybe also Q2).
Besides, it is not very likely, that one day PROPOSAl will calculate the real secondary particles of an hadronic cascade. Therefore, other calculation tools exist. The CORSIKA Problem should be dealt in CORSIKA.

The same of course also holds for the photo-hadronic, or weak interactions, or simply where a hadronic cascade is created.

Reply:
Should PROPOSAL still issue a warning after we implemented the hadronic pseudoparticle / hadronic cascade into the Photonuclear secondaries calculator?

Reply:
They may be annoying, but I can also understand if they are left to signal the user, that not a real particle ws produced.

Reply:
Ah, and I see, that my other two concerns should be handled in another PR since
- for the weak interaction the secondary calculator is not yet fully implemented
- the photon-hadron interaction is not implemented at all
these are more relevat for a milestone 7.1.0